frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/result-tab/result-tab-row.scss

```diff
@@ -0,0 +1,17 @@
+.kubevirt-create-vm-modal___result-tab-row-container {
```

Comment:
Are we writing specific namespace based classes for Kubevirt? I'm trying to learn more about the general css approach, is there any documentation on this ?

Reply:
yes, since this component is kubevirt specific. The components are moved to core once they are used by multiple plugins. (there is some doc [here](https://github.com/openshift/console/blob/master/STYLEGUIDE.md))

Reply:
+1 thanks for that link!